### ISSUE TITLE
Bigger hit area for editing a label

### ISSUE THREAD

**Issue by laurakalbag** [statelyai/feedback#210]
### Details
The hit area should be bigger so that double-clicking anywhere in the label area focuses the input.
### Category

Editor

---
Originally posted on: 2022-06-25T19:07:32.967Z
By: Tejas Kumar
Previous votes: 4
Previous status: open
